### DISCUSSION TITLE
What is 39955 port?

### DISCUSSION THREAD

**Discussion by d47081** [zeronet-conservancy/zeronet-conservancy#268] [Q&A] [answered]
Auto generated in config `fileserver_port = 39955`

Must I open it?

**Comment by slrslr** [comment, marked as answer]
At this port Zeronet is listening for incoming connections from other peers.
You do not have to "open" it, but you should (same as with a bittorrent file sharing port), in order to connect more peers (ones that do not have this port open/forwarded).

**Reply by d47081** [reply]
Thank you!
